ui/pages/confirm-add-suggested-token/index.scss

```diff
@@ -49,4 +45,47 @@
     margin-right: 12px;
     flex: 0 0 auto;
   }
+
+  // overrides
```

Comment:
this comment might lose context soon, can we remove? Unless it's an important note.

Reply:
updated https://github.com/MetaMask/metamask-extension/pull/31454/commits/9ea820a53f2855b96a2f0e9e48a124ffed9e3adb